event_routing_backends/processors/tests/fixtures/current/edx.course.enrollment.activated.enterprise.json

```diff
@@ -0,0 +1,24 @@
+{
+    "context": {
+        "accept_language": "en-US,en;q=0.9",
+        "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36",
+        "client_id": null,
+        "course_id": "course-v1:edX+DemoX+Demo_Course",
+        "host": "localhost:18000",
+        "ip": "172.18.0.1",
+        "org_id": "edX",
```

Comment:
we should pass in `enterprise_uuid` here to cater for the enterprise use case and in in expected data we should assert it in context.

Reply:
done